pandas/tests/reshape/test_pivot.py

```diff
@@ -896,12 +896,6 @@ def _check_output(
             totals = table.loc[("All", ""), value_col]
             assert totals == self.data[value_col].mean()
 
-        # no rows
-        rtable = self.data.pivot_table(
-            columns=["AA", "BB"], margins=True, aggfunc=np.mean
-        )
-        assert isinstance(rtable, Series)
-
```

Comment:
this gives a wrong shape, should not return Series but a DataFrame with AA and BB for columns, I created an issue to address it!